src/Components/Scenes/RequestAssets/RequestAsset/RequestAsset-Container.js

```diff
@@ -1,10 +1,17 @@
 import { connect } from 'react-redux';
-import RequestAsset from './RequestAsset';
+import RequestTable from './RequestAsset';
```

Comment:
nitpick: should this be `RequestAsset` instead? I don't see anything referenced as `RequestTable` anywhere else?

Reply:
Changed